Ports/libjxl/package.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port='libjxl'
+version='2023.09.11'
+files=(
+    "git+https://github.com/libjxl/libjxl#ff8a9c1cd54d324137c1905017542ffedeacdef8"
+)
+
+depends=(
+    'brotli'
+    'highway'
+    'lcms2'
+    'libpng'
+)
+
+workdir='libjxl'
+
+configopts=(
+    "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DBUILD_TESTING=OFF"
+    "-DJPEGXL_ENABLE_BENCHMARK=OFF"
+    "-DJPEGXL_ENABLE_SKCMS=OFF"
+    "-DJPEGXL_ENABLE_SJPEG=OFF"
+)
+useconfigure='true'
+
+configure() {
+    run git submodule update --init --recursive
```

Comment:
We should just add `--recurse-submodules` to the `checkout` command in `.ports_include.sh`. The ports script shouldn't bother with this, much less the `configure` step.

Reply:
Thanks for fixing that!
I really don't have time these days, I appreciated that you took care of that.